MoreLinq.Test/SkipLastWhileTest.cs

```diff
@@ -0,0 +1,112 @@
+﻿#region License and Terms
```

Comment:
nit: There's a UTF-8 BOM marker at the beginning of this file [that shouldn't be there](https://github.com/morelinq/MoreLINQ/blob/1726b52594c93e342bb629fb7652d50426815851/.editorconfig#L23).